warehouse/query-core/src/main/java/datawave/query/jexl/functions/EvaluationPhaseFilterFunctions.java

```diff
@@ -310,6 +307,72 @@ public static boolean excludeRegex(Iterable<?> values, String regex) {
         return FunctionalSet.unmodifiableSet(matches);
     }
     
+    private static class NaturalComparator<T extends Comparable<T>> implements Comparator<T> {
```

Comment:
I did this quickly back in April but I'll replace this with a lib function.  As memory serves there is a natural comparator in commons ComparatorUtils